code/linkfield/Extensions/LinkPageExtension.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace SilverStripe\FrameworkTest\LinkField\Extensions;
+
+use SilverStripe\Forms\FieldList;
+use SilverStripe\LinkField\Models\Link;
+use SilverStripe\LinkField\Form\LinkField;
+use SilverStripe\LinkField\Form\MultiLinkField;
+use SilverStripe\LinkField\Models\EmailLink;
+use SilverStripe\LinkField\Models\PhoneLink;
+use SilverStripe\LinkField\Models\SiteTreeLink;
+use SilverStripe\ORM\DataExtension;
+
+class LinkPageExtension extends DataExtension
```

Comment:
```suggestion
use SilverStripe\ORM\Extension;
use SilverStripe\Dev\TestOnly;

class LinkPageExtension extends Extension implements TestOnly
```
DataExtension will be removed in a future version

Reply:
Updated